.circleci/create_circleci_config.py

```diff
@@ -154,6 +154,7 @@ def job_name(self):
         "sudo apt-get -y update && sudo apt-get install -y libsndfile1-dev espeak-ng time",
         "pip install --upgrade pip",
         "pip install .[sklearn,torch,testing,sentencepiece,torch-speech,vision,timm]",
+        "pip install natten",
```

Comment:
Let's remove this from here. After internal dicussion, we will test this model with LayoutLM and make this job the test for all "exotic" models that require additional dependencies. So please add it to `layoutlm_job` below :-)

Reply:
Done.